client/state/reader/lists/reducer.js

```diff
@@ -97,7 +97,7 @@ export const subscribedLists = withSchemaValidation(
 export const updatedLists = withSchemaValidation( updatedListsSchema, ( state = [], action ) => {
 	switch ( action.type ) {
 		case READER_LIST_UPDATE_SUCCESS:
-			const newListId = get( action, 'data.list.ID' );
+			const newListId = action?.data?.list?.ID;
```

Comment:
It's not possible that `action` will be `null` / `undefined` here - if we get to this `switch` case, `action.type` will be equal to `READER_LISTS_FOLLOW_SUCCESS `.